.github/blunderbuss.yml

```diff
@@ -59,7 +59,6 @@ assign_issues_by:
   - m-strzelczyk
 - labels:
   - 'api: container'
-  - 'api: texttospeech'
```

Comment:
Could you move this down under `python-samples-reviewers` batch?

Reply:
sure thing!